<!DOCTYPE UI>
<ui version="4.0" >
 <class>configDlg</class>
 <widget class="QWidget" name="configDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>342</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Devices</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>11</number>
   </property>
   <property name="topMargin" >
    <number>11</number>
   </property>
   <property name="rightMargin" >
    <number>11</number>
   </property>
   <property name="bottomMargin" >
    <number>11</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
    </layout>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>190</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="1" column="0" >
    <widget class="QTableWidget" name="tblDevices" >
     <property name="minimumSize" >
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="showGrid" >
      <bool>true</bool>
     </property>
     <property name="numRows" stdset="0" >
      <number>0</number>
     </property>
     <property name="numCols" stdset="0" >
      <number>4</number>
     </property>
     <column>
      <property name="text" >
       <string>device</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>label</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Input</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Output</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="KPushButton" native="1" name="bAdd" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" stdset="0" >
        <string>Add</string>
       </property>
       <property name="shortcut" stdset="0" >
        <string/>
       </property>
       <property name="stdItem" stdset="0" >
        <number>27</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" native="1" name="bRemove" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" stdset="0" >
        <string>Remove</string>
       </property>
       <property name="shortcut" stdset="0" >
        <string/>
       </property>
       <property name="stdItem" stdset="0" >
        <number>28</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QWidget</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kpushbutton.h</include>
  <include location="local" >kpushbutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
